wellcomeml/ml/cnn.py

```diff
@@ -233,11 +240,17 @@ def residual_attention(x1):
         output_activation = (
             "sigmoid" if nb_outputs == 1 or self.multilabel else "softmax"
         )
+        if self.feature_approach == "multilabel-attention":
+            # each output comes from an attention head
+            nb_outputs = 1
         out = tf.keras.layers.Dense(
             nb_outputs,
```

Comment:
I don't understand, if this is multilabel attention, the dense layer will only have one unit?

Reply:
Ah, I think I worked out the maths. The layer will be `(batch_size, number_of_labels, 1)`

Reply:
👌